code/modules/events/pirates.dm

```diff
@@ -14,56 +14,21 @@
 	return ..()
 
 /datum/round_event/pirates
-	startWhen = 60 //2 minutes to answer
-	var/datum/comm_message/threat
-	var/payoff = 0
-	var/payoff_min = 20000
-	var/paid_off = FALSE
+	//honk -- remove announcement
```

Comment:
The marker comment usage is inconsistent in this file. Could you switch to using comments in the form of
```
/* honk start -- reason
<stuff you want gone>
honk end /*
```
instead of deleting the chucks of code outright?